app/views/reports/regions/_patient_breakdown_charts.html.erb

```diff
@@ -41,8 +41,8 @@
                   <%= number_with_delimiter(@chart_data[:ltfu_trend].last_value(:cumulative_assigned_patients)) %>
                 </span>
                 patients registered till
-                <span data-registrations-period-end="<%= @chart_data[:ltfu_trend].last_value(:period_info)["start_date"] %>">
-                  <%= @chart_data[:ltfu_trend].last_value(:period_info)["start_date"] %>
+                <span data-registrations-period-end="<%= @period.bp_control_range_end_date %>">
+                  <%= @period.bp_control_range_end_date %>
```

Comment:
Does this ensure that the "registered until" date on the breakdown charts always matches the one in the top-4 cards?